docs/understanding-tracking-design/managing-tracking-scenarios-console/index.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "🆕 Managing tracking scenarios in the console"
+date: "2020-04-17"
+sidebar_position: 95
+---
+
```

Comment:
I copied the banner from the data-structures-builder and rephrased a little. However the link for the `Managing tracking scenarios with API` depends on the other PR https://github.com/snowplow/documentation/pull/441. Any suggestion? Is it acceptable to set this TODO or do you think it is better to create a new PR with the api and console documentation?

Reply:
I don’t think it’s true that UI is only available in Cloud 🤔

For this PR, I suggest that we just keep “Only available in BDP”. Once this one is merged, you can update the other PR to link to the API docs and vice versal.

Reply:
> I don’t think it’s true that UI is only available in Cloud 🤔

@cksnp can you give us some light about this 😄 

> For this PR, I suggest that we just keep “Only available in BDP”. Once this one is merged, you can update the other PR to link to the API docs and vice versa.

👍

Reply:
@stanch, @cksnp and I had some thinking around this and we would like to validate with Elisa when she is back.

Reply:
Tracking Scenario will be available through Console and the API to Enterprise customer. 
I've asked the team to check effort to release to Cloud customers, but it will still be both UI and API.

I don't think we have the capability to enable UI only or API only, as they share the auth token? I could be wrong.